src/rpc/client.h

```diff
@@ -8,6 +8,12 @@
 
 #include <univalue.h>
 
+struct RPCConvertParam
+{
+    char methodName[128]; //!< method whose params want conversion
```

Comment:
Nit: Any particular reason for this?

You can just keep the `std::string`, so it is move-only:

```diff
diff --git a/src/rpc/client.cpp b/src/rpc/client.cpp
index 9d4e788..553c21b 100644
--- a/src/rpc/client.cpp
+++ b/src/rpc/client.cpp
@@ -23,3 +23,3 @@ using namespace std;
 
-static const struct RPCConvertParam vRPCConvertParams[] =
+static const CRPCConvertParam vRPCConvertParams[] =
 {
@@ -79,3 +79,3 @@ CRPCConvertTable::CRPCConvertTable()
     for (unsigned int i = 0; i < ARRAYLEN(vRPCConvertParams); i++) {
-        members.insert(std::make_pair(std::string(vRPCConvertParams[i].methodName),
+        members.insert(std::make_pair(vRPCConvertParams[i].methodName,
                                       vRPCConvertParams[i].paramIdx));
@@ -84,3 +84,3 @@ CRPCConvertTable::CRPCConvertTable()
     for (unsigned int i = 0; i < ARRAYLEN(walletRPCConversions); i++) {
-        members.insert(std::make_pair(std::string(walletRPCConversions[i].methodName),
+        members.insert(std::make_pair(walletRPCConversions[i].methodName,
                                       walletRPCConversions[i].paramIdx));
diff --git a/src/rpc/client.h b/src/rpc/client.h
index 3140f59..9b707ee 100644
--- a/src/rpc/client.h
+++ b/src/rpc/client.h
@@ -10,6 +10,7 @@
 
-struct RPCConvertParam
+class CRPCConvertParam
 {
-    char methodName[128]; //!< method whose params want conversion
-    int paramIdx; //!< 0-based idx of param to convert
+public:
+    std::string methodName; //!< method whose params want conversion
+    int paramIdx;           //!< 0-based idx of param to convert
 };
diff --git a/src/wallet/rpcjsonconversion.h b/src/wallet/rpcjsonconversion.h
index cf5d919..d6c331d 100644
--- a/src/wallet/rpcjsonconversion.h
+++ b/src/wallet/rpcjsonconversion.h
@@ -7,3 +7,5 @@
 
-static const struct RPCConvertParam walletRPCConversions[] =
+class CRPCConvertParam;
+
+static const CRPCConvertParam walletRPCConversions[] =
 {